src/components/ToastTextToCad.tsx

```diff
@@ -533,23 +568,38 @@ export function ToastPromptToEditCadSuccess({
             onClick={() => {
               sendTelemetry(modelId, 'accepted', token).catch(reportRejection)
               toast.dismiss(toastId)
-
-              // Write new content to disk since they have accepted.
-              codeManager
-                .writeToFile()
-                .then(() => {
-                  // no-op
-                })
-                .catch((e) => {
-                  console.error('Failed to save prompt-to-edit to disk')
-                  console.error(e)
-                })
+              /**
+               * NO OP. Do not rewrite code to disk, we already do this ahead of time. This will dismiss the toast.
+               * All of the files were already written! Don't rewrite the current code editor.
+               * If this prompt to edit makes 5 new files, the code manager is only watching 1 of these files, why
+               * would it rewrite the current file selected when this is completed?
```

Comment:
nit: I don't think this last sentence in the comment will be very helpful to future devs reading through this code @nadr0 
```suggestion
```

Reply:
Yeah we can delete it. I was thinking out loud and left this for the future. I was trying to leave something to say "think about 1 file vs n files" because that is something we need to handle in different parts of the system. You can commit that suggestion to delete it.